bin/main.rb

```diff
@@ -1,44 +1,21 @@
 #!/usr/bin/env ruby
-board = %w[1 2 3 4 5 6 7 8 9]
-puts "#{board[0]} | #{board[1]} | #{board[2]}"
-puts "#{board[3]} | #{board[4]} | #{board[5]}"
-puts "#{board[6]} | #{board[7]} | #{board[8]}"
+require './lib/gametictac'
```

Comment:
- [ ] [OPTIONAL] Using `require_relative` for files in your directory instead of `require` is a ruby convention.